src/WorkflowManager/appsettings.Development.json

```diff
@@ -39,9 +39,9 @@
     "messaging": {
       "publisherServiceAssemblyName": "Monai.Deploy.Messaging.RabbitMQ.RabbitMQMessagePublisherService, Monai.Deploy.Messaging.RabbitMQ",
       "publisherSettings": {
-        "endpoint": "localhost",
-        "username": "admin",
```

Comment:
can you undo these changes for development?

Reply:
Revert the changes.